package.json

```diff
@@ -183,6 +188,7 @@
     "@testing-library/react": "^10.4.9",
     "@testing-library/user-event": "^12.2.0",
     "commander": "^3.0.2",
+    "cypress": "4.10.0",
```

Comment:
Could you try with the last version -> 6.4.0?

Reply:
does not work :(